src/main/java/eu/elision/marketplace/web/dtos/Pair.java

```diff
@@ -0,0 +1,15 @@
+package eu.elision.marketplace.web.dtos;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
+public class Pair<A, V> {
```

Comment:
`Pair` is an existing Spring class already. Also, this pair represents an attribute value pair. Try using a more specific name for this class. (AttributePair, AttributeValue, ...?)